app/views/admin/talentlms_courses/_form.haml

```diff
@@ -0,0 +1,6 @@
+= f.input :config_id, as: :select_two, label: 'Subdomain', collection: Talentlms::Config.collection_for_form, selected: @course&.config&.id
+= f.input :name
+= f.input :courseid, label: 'Course ID'
+- label = if Rails.env.production? then 'Months' else 'Days' end
+= f.input :months_to_expiration, label: "#{label} Until Required Recertification", hint: 'Leave blank if recertification is not required'
```

Comment:
Can we dry up this months/days calculation and just ask the Facade class for it?  I believe there is something similar in there, but might as well just make constant.

Reply:
Cleaned this up [here](https://github.com/greenriver/hmis-warehouse/pull/4858/commits/c892c7caa76b3b9252e0fd914a0f04432ab6dfa8)